rust/sealevel/programs/hyperlane-sealevel-token-collateral/Cargo.toml

```diff
@@ -30,6 +30,7 @@ hyperlane-sealevel-igp = { path = "../hyperlane-sealevel-igp", features = [
 ] }
 hyperlane-sealevel-message-recipient-interface = { path = "../../libraries/message-recipient-interface" }
 hyperlane-sealevel-token-lib = { path = "../../libraries/hyperlane-sealevel-token" }
+hyperlane-warp-route = { path = "../../../main/applications/hyperlane-warp-route" }
```

Comment:
i think we want to completely avoid depending on the `main` workspace so we can eventually decouple the two workspaces and fix dependency issues

you could create a new crate that has no dependencies on the two workspaces, but is depended on by both and defines `TokenMessage`

Reply:
Discussed in person. We won't separate it at this point.